sample/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.regnosys.rosetta.code-generators</groupId>
+        <artifactId>parent</artifactId>
+        <version>0.0.1</version>
```

Comment:
`<artifactId>parent</artifactId>` is this correct?

Reply:
yes!